cvat-core/src/storage.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2022 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+
+(() => {
+    const { ArgumentError } = require('./exceptions');
+    const { StorageLocation } = require('./enums');
+
+    /**
+     * Class representing a storage for import and export resources
+     * @memberof module:API.cvat.classes
+     * @hideconstructor
+     */
+    class Storage {
+        location: typeof StorageLocation;
+        cloudStorageId: number;
```

Comment:
If redefine StorageLocation as ``enum``:

```suggestion
        public location: StorageLocation;
        public cloudStorageId: number;
```